lms/static/sass/features/_course-upgrade-message.scss

```diff
@@ -0,0 +1,118 @@
+/*
+  NOTE: If you make significant changes to the design, remember to update the Segment event properties and change
+  the creative property. This will allow us to better track individual performance of each style of the message.
+  Search for the courseware_verified_certificate_upsell promotion ID.
+ */
+$upgrade-message-background-color: #005686;
+$upgrade-message-button-color: #008100;
+
+// Expanded upgrade message
+.vc-message {
+  background: $upgrade-message-background-color;
+  color: white;
+  padding: 20px;
+  position: relative;
+  margin: 0 0 20px;
+  // CSS animation for smooth height transition
+  -webkit-transition: all 0.2s ease-in-out;
+  transition: all 0.2s ease-in-out;
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+  // Message copy
+  h3 {
+    font-size: 1.25rem;
+    font-weight: 700;
+    margin-bottom: 1rem;
+    float: left;
```

Comment:
Use the bi-app sass functions so that this will work with right-to-left, i.e.

    @include float(left);

You should use these for all usages of left or right in this file.